prql-compiler/prqlc/tests/test.rs

```diff
@@ -112,13 +112,13 @@ fn compile_help() {
 
     -t, --target <TARGET>
             Target to compile to
-            
```

Comment:
This change from pre-commit-ci seems made the test fail on the main branch.
https://github.com/PRQL/prql/actions/runs/5473711576/jobs/9967543314

@max-sixty Any idea to disable this?

Reply:
Sorry, I guess I missed this on vacation. Thanks for the follow-up to fix...